features/Steps/ResponseSteps.php

```diff
@@ -308,4 +321,42 @@ private function removeDates(string $value): string
         $datePattern = '/(?<=dcterms:created\s|dcterms:modified\s)"\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\+\d{2}:\d{2}"/';
         return preg_replace($datePattern, '""', $value);
     }
+
+    /**
+     * @Then I calculate the image hash with description :description, copyright :copyrightHolder and language :language for :imageId
+     */
+    public function iCalculateTheImageHashWith(string $description, string $copyrightHolder, string $language, string $imageId): void
```

Comment:
Is it not possible to pass the actual image id?

Reply:
If I pass things with %{imageId} it just contains the literal string. I don't know how to fix this.

Reply:
I think that is part of the `FeatureContext` class, that class contains `@Transform` methods.